src/main/java/com/odysseusinc/arachne/datanode/controller/admin/BaseAdminController.java

```diff
@@ -133,4 +158,109 @@ public JsonResult checkAtlasConnection(@PathVariable("id") Long id) {
         return result;
     }
 
+    @ApiOperation(value = "Invalidate all unfinished analyses")
```

Comment:
Shouldn't this method be in AnalysisController?

Reply:
May be, leaved as is just for the same reasons as above.